README.md

```diff
@@ -28,13 +28,15 @@ Requirements
 Installation
 --------------
 
-This application can be installed and run by following the steps below:
+This application can be installed by following the steps below:
 
     $ git clone https://github.com/unt-libraries/m2m.git
 
     $ cd m2m
 
-    $ python setup.py install
+    $ pip install .
```

Comment:
You didn't have to change this, btw. It should now work fine with _either_ installation method. Use whichever you want